sass/_base.scss

```diff
@@ -314,7 +235,7 @@ pre table {
     background-repeat: no-repeat;
     padding: 1rem;
 
-    max-width: 45%;
+    max-width: 23rem;
```

Comment:
Is this really going to cause the same effect, or is there some other reason behind this change?

Reply:
The reason is that if the card is used somewhere that is super wide, the card will be very stretched. In practice 23rem "feels" nice as a maximum width